CONTRIBUTING.md

```diff
@@ -159,4 +161,4 @@ The requirement is simply that a date-based frequency be used so that restarts a
 
 **sync**
 
-This should not be enabled. Nor should `sync_path` be set to a real path. Ideally set `sync_path` to `null`.
\ No newline at end of file
+This should not be enabled. Nor should `sync_path` be set to a real path. Ideally set `sync_path` to `null`.
```

Comment:
Maybe use the following as an example as it's `path` rather than `sync_path`
```
sync:
    enable: false
    path: null
```

Reply:
Thanks.